src/boot/lib/ast.ml

```diff
@@ -112,9 +112,9 @@ and const =
 | Cexit
 (* MCore Symbols *)
 | CSymb of int
-| Cgensym
+| CgenSym
```

Comment:
I we convert this to camel case, then I feel like it should probably be `symGen` instead.

Reply:
Not sure about this one. Gensym is more of a word in PL and I would prefer to keep it as one word, i.e. `gensym`.

Reply:
This change is now reverted, as discussed during the meeting